ui/raidboss/data/07-dt/alliance/jeuno-first-walk.txt

```diff
@@ -0,0 +1,750 @@
+### JEUNO: THE FIRST WALK
+# ZoneId: 1248
+
+hideall "--Reset--"
+hideall "--sync--"
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# Prishe Of The Distant Chains #
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 9FE6 9FF1 9FF3 9FEB 9FEC 9FED A003 9FF5 9FFE 9FFF
+
+# A number of Prishe's abilities are two-part,
+# with a visual snapshot occurring before a damage event.
+# The snapshot locks in when the visual element resolves,
+# so we display the visual and then ignore the damage event.
+
+# The grand dais will be sealed off
+10000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "141E" } window 10000,1
+10008.1 "Banishga" Ability { id: "9FE7", source: "Prishe of the Distant Chains" } window 10008.1,10
```

Comment:
The start of the timeline was very delayed syncing here, to the point that the ability went off quite a bit for the timeline indicated.  I will need to try a few more runs to see if it's a consistent issue.

Reply:
For comparison, my one run of Jeuno had this:
```
41|2024-11-16T17:48:34.3280000-05:00|800375C0|7DC|
20|2024-11-16T17:48:40.5170000-05:00|40000610|Prishe of the Distant Chains|9FE7|Banishga|
22|2024-11-16T17:48:45.5090000-05:00|40000610|Prishe of the Distant Chains|9FE7|Banishga|
```

So that would be something like:
```
# The grand dais will be sealed off
10000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "141E" } window 10000,1
10006.1 "--sync--" StartsUsing { id: "9FE7", source: "Prishe of the Distant Chains" } window 10006.1,10
10011.1 "Banishga" Ability { id: "9FE7", source: "Prishe of the Distant Chains" }
#adjust the rest of the timeline +3s
```

Reply:
I suspect that what happened here is that I started with a base timeline from FFLogs, since I haven't personally seen the encounter go longer than six minutes. FFLogs doesn't include zone seals, so I forgot to correct for that.